changelog.md

```diff
@@ -1,3 +1,32 @@
+# 4.2.0 / YYYY-MM-DD
+
+### New Features
+
+- Handle right-to-left (rtl) languages on interface
+- Introduce arab language selection, and add arabic translation via Weblate
+- Notification can be marked as read without redirection
+- UI/UX improvement (responsive, buttons, redirections)
+
+### Fixed Issues
+
+- Backend: [#5559](https://github.com/tracim/tracim/issues/5559),
+[#5378](https://github.com/tracim/tracim/issues/5378)
+- Frontend: [#5313](https://github.com/tracim/tracim/issues/5313),
```

Comment:
If no more issue in frontend list, `,` is not necessary (I think more issue need to be added here ^^ )